test/integration/sequelize.test.js

```diff
@@ -265,6 +266,24 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
           expect(logger.args[0][0]).to.be.match(/Executed \(default\): select 1; Elapsed time: \d+ms/);
         });
       });
+    
+      it('logs warnings when there are warnings', function() {
+        var logger = sinon.spy();
+        logger.reset();
+        var sequelize = Support.createSequelizeInstance({
+          logging: logger,
+          benchmark: false
+        });
+        var insertWarningQuery = 'INSERT INTO ' + qq(this.User.tableName) + ' (username, email_address, ' +
+          qq('createdAt') + ', ' + qq('updatedAt') +
+          ") VALUES ('john', 'john@gmail.com', 'HORSE', '2012-01-01 10:10:10')";
+
+        return sequelize.query(insertWarningQuery)
+        .then(function(results) {
+          expect(logger.calledTwice).to.be.true;
+          expect(logger.args[1][0]).to.be.match(/^MySQL Warnings \(default\): Data truncated for column 'createdAt' at row \d+$/m);
```

Comment:
The error message is slightly different on the mysql version on travis, so this test is failing